products/veeam-backupandreplication

```diff
@@ -0,0 +1,111 @@
+---
+title: Veeam Backup & Replication
+category: app
+tags: veeam
+iconSlug: veeam
+permalink: /veeam-backupandreplication
+alternate_urls:
+-   /veeambackupandreplication
+releasePolicyLink: https://www.veeam.com/product-lifecycle.html
+activeSupportColumn: End of Fix
+releaseColumn: false
+releaseDateColumn: true
+eolColumn: End of support
+
+releases:
+-   releaseCycle: "12.0"
+    releaseDate: 2023-02-01
+    support: true
+    eol: 2026-02-01
+    link: "https://www.veeam.com/kb4420"
+
+-   releaseCycle: "11.0"
+    releaseDate: 2021-02-01
+    support: 2023-02-01
+    eol: 2024-02-01
+    link: null
+
+-   releaseCycle: "10.0"
+    releaseDate: 2020-02-01
+    support: 2021-02-01
+    eol: 2023-02-01
+    link: null
+
+-   releaseCycle: "9.5"
+    releaseDate: 2016-01-01
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "9.0"
+    releaseDate: 2016-01-01
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "8.0"
+    releaseDate: 2014-01-01
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "7.0"
+    releaseDate: 2013-01-01
+    support: false    
+    eol: true
+    link: null
+
+-   releaseCycle: "6.5"
+    releaseDate: 2012-01-01
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "6.0"
+    releaseDate: 2011-01-01
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "5.0"
+    releaseDate: 2010-01-01
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "4.0"
+    releaseDate: 2009-10-29
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "3.0"
+    releaseDate: 2009-01-01
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "2.0"
+    releaseDate: 2008-07-30
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "1.0"
+    releaseDate: 2008-02-26
+    support: false
+    eol: true
+    link: null
+
+
+---
+
+> [Veeam Backup & Replication](https://helpcenter.veeam.com/docs/backup/qsg_vsphere/about_vbr.html?ver=120) is a data protection and disaster recovery solution for virtual, physical and cloud environments.
+
+## [End of Fix](https://www.veeam.com/product-lifecycle.html)
```

Comment:
No need to use link in titles: the presence of a `releasePolicyLink` will trigger the generation of short paragraph for it.

For example for Ruby this is:

> More information is available on the [Ruby website](https://www.ruby-lang.org/en/downloads/releases/).